components/kyma-environment-broker/Makefile

```diff
@@ -50,3 +50,19 @@ testing-with-database-network:
 
 clean-up:
 	@docker network rm $(TESTING_DB_NETWORK) || true
+
+run-local:
+	kind create cluster --name compass --config config/kind/cluster.yaml
+	kubectl apply -f https://projectcontour.io/quickstart/contour.yaml
+	kubectl patch daemonsets -n projectcontour envoy -p '{"spec":{"template":{"spec":{"nodeSelector":{"ingress-ready":"true"},"tolerations":[{"key":"node-role.kubernetes.io/master","operator":"Equal","effect":"NoSchedule"}]}}}}'
+	while [[ `kubectl get pods -n projectcontour -l app=envoy -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}'` != "True" ]]; do \
+  		echo "Waiting for envoy pod" && sleep 2;\
+	done;
+	ko apply -f config
+	while [[ `kubectl get pods -n compass-system -l app=postgresql -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}'` != "True" ]]; do \
+  		echo "Waiting for postgresql pod" && sleep 2;\
```

Comment:
There seems to be permission problem with the database. Is it still working for you?

```shell
$ k logs -n compass-system compass-postgresql-0
 11:00:11.28
 11:00:11.29 Welcome to the Bitnami postgresql container
 11:00:11.29 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-postgresql
 11:00:11.29 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-postgresql/issues
 11:00:11.29 Send us your feedback at containers@bitnami.com
 11:00:11.29
 11:00:11.30 INFO  ==> ** Starting PostgreSQL setup **
 11:00:11.33 INFO  ==> Validating settings in POSTGRESQL_* env vars..
 11:00:11.34 INFO  ==> Initializing PostgreSQL database...
 11:00:11.34 INFO  ==> postgresql.conf file not detected. Generating it...
 11:00:11.35 INFO  ==> pg_hba.conf file not detected. Generating it...
 11:00:11.36 INFO  ==> Deploying PostgreSQL with persisted data...
 11:00:11.36 INFO  ==> Configuring replication parameters
 11:00:11.38 INFO  ==> Configuring fsync
 11:00:11.38 INFO  ==> Loading custom scripts...
 11:00:11.39 INFO  ==> Loading user's custom files from /docker-entrypoint-initdb.d ...
 11:00:11.39 INFO  ==> Starting PostgreSQL in background...
/tmp:5432 - accepting connections
touch: cannot touch '/bitnami/postgresql/.user_scripts_initialized': Permission denied
 11:00:11.53 INFO  ==> Stopping PostgreSQL...
```

```shell
$ k describe pod -n compass-system compass-postgresql-0

Warning  BackOff           62s (x25 over 5m36s)   kubelet, compass-worker  Back-off restarting failed container
```